file-formats/aifa/type/zif_aff_aifa_v1.intf.abap

```diff
@@ -0,0 +1,103 @@
+INTERFACE zif_aff_aifa_v1
+   PUBLIC.
+
+  "! $values { @link zif_aff_aifa_v1.data:co_error_handling }
+  "! $default { @link zif_aff_aifa_v1.data:co_error_handling.ignore_data }
+  TYPES ty_error_handling_type TYPE c LENGTH 1.
+  CONSTANTS:
+    BEGIN OF co_error_handling,
+      "! <p class="shorttext">Treat As Error</p>
+      "! Treat as error if check is not successful
+      trigger_error TYPE c LENGTH 1 VALUE '',
```

Comment:
Text and file name do not match. I guess I would try avoid to use "trigger" if possible since it shall be avoided according to https://help.sap.com/docs/TERMINOLOGY/25cbeaaad3c24eba8ea10b579ce81aa1/83a23df24013403ea4c1fdd0107cc0fd.html

However, "Treat as Error" or "Raise Error" should be fine.

```suggestion
      "! <p class="shorttext">Treat as Error</p>
      "! Treat as error if check is not successful
      treat_as_error TYPE c LENGTH 1 VALUE '',
```

```suggestion
      "! <p class="shorttext">Raise Error</p>
      "! Raise error if check is not successful
      raise_error TYPE c LENGTH 1 VALUE '',
```